### ISSUE TITLE
5.x: Preview does not respect LTR setting

### ISSUE THREAD

**Issue by bartkamphorst** [gollum/gollum#1414]
Changing the new RTL language option in Edit does not change the language direction in the page Preview.

**Comment by dometto** [comment]
It should also be possible to load the preview in a separate tab or page. Currently trying to do that just takes you to the edit page.

**Comment by bartkamphorst** [comment]
> It should also be possible to load the preview in a separate tab or page. 

Why should it? Right-clicking the Preview tab on github also does not show a 'Open in new tab' option.

**Comment by dometto** [comment]
I think this would be nice functionality: suppose you want to walk through the source document to fix mistakes, while having a preview open in a separate window.

**Comment by dometto** [comment]
It would also be nice if the AJAX call was only done if anything was actually edited.

**Comment by dometto** [comment]
> 

    It should also be possible to load the preview in a separate tab or page.

> Why should it? Right-clicking the Preview tab on github also does not show a 'Open in new tab' option.

Maybe that idea is too complicated, but perhaps instead it would be nice if you could switch between the tabs using hotkeys? If so, we should make sure to document it [here](https://github.com/gollum/gollum/wiki/Keyboard-Shortcuts)

**Comment by bartkamphorst** [comment]
> It would also be nice if the AJAX call was only done if anything was actually edited.

Agreed. However, I think we should leave that as an improvement issue.

> perhaps instead it would be nice if you could switch between the tabs using hotkeys?

Sure, I'm all for it, but it's not a showstopper for me for 5.0.
